cpp/src/io/parquet/predicate_pushdown.cpp

```diff
@@ -139,82 +75,8 @@ struct stats_caster {
     if constexpr (cudf::is_compound<T>() && !std::is_same_v<T, string_view>) {
       CUDF_FAIL("Compound types do not have statistics");
     } else {
-      // Local struct to hold host columns
```

Comment:
Declared in `stats_filter_helpers.hpp` and defined at the bottom of this file now.